server/server.go

```diff
@@ -26,6 +25,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/open-policy-agent/opa/bundle"
```

Comment:
[nit] shouldn't this go to the bottom, and the `github.com/gorilla/mux` import be put into its own block?

Reply:
Kind of fixed. For some reason running goimports on server.go doesn't put the import into it's own block. I'd rather just let tool(s) enforce the import ordering to whatever extent they do.